src/components/shared/radio/radio.js

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import { hot } from "react-hot-loader";
+import "./radio.scss";
+import PropTypes from "prop-types";
+
+
+class Radio extends React.Component {
+    static propTypes = {
```

Comment:
I'd rather stick to one kind of declaring propTypes

Reply:
Fixed